undetected_chromedriver/__init__.py

```diff
@@ -19,31 +19,31 @@
 """
 
 
-__version__ = "3.1.5r4"
+__version__ = "3.1.5r5"
 
 
+import inspect
 import json
 import logging
 import os
 import re
 import shutil
 import sys
 import tempfile
-import time
-import inspect
 import threading
+import time
 
 import selenium.webdriver.chrome.service
 import selenium.webdriver.chrome.webdriver
 import selenium.webdriver.common.service
 import selenium.webdriver.remote.webdriver
+from selenium.webdriver.chrome.service import Service
```

Comment:
please change this to `import selenium.webdriver.chrome.service `
and use it later selenium.webdriver.common.service.Service() to create your service.